README.md

```diff
@@ -57,6 +57,8 @@ At the very least, you should set the right type of board in config.h, most othe
 
 When using the US_915 you need to select which sub-band you will use, by default it is sub-band 6.
 
+We have to manually set the frequency plan (EU_868, US_915, AS_923, AS_923_2...) in the Config.h of the library.
```

Comment:
Also you can define the frequency plan before `#include <lorawan.h>`

For example:

```c++
#define US_915

#include <lorawan.h>

void setup() {
   // code
}

void loop() {
   // code
}
```

Reply:
I tried already.
It didn't seem to work.

May be the files of the library only get the "define" in the library only.

Reply:
I will try to define again in the main Arduino.

Is it a hint that "define" frequency plan in the main Arduino may not affect the frequency plan the library?

From README:
A number of features can be configured or disabled by editing the config.h file in the library folder. Unfortunately the Arduino environment does not offer any way to do this (compile-time) configuration from the sketch, so be careful to recheck your configuration when you switch between sketches or update the library.

Reply:
OK, I tried that multiple times on regular C/C++ projects and it works, so I thought it will work on Arduino, but apparently it doesn't.

Reply:
Ok, thanks a lot for your confirmation.

In the meantime, I add a note in README to help new users on this.